runtime/tests/interpreter/path_test.go

```diff
@@ -56,3 +59,86 @@ func TestInterpretPath(t *testing.T) {
 		})
 	}
 }
+
+func TestConvertStringToPath(t *testing.T) {
```

Comment:
Name differs from what is on master:

```suggestion
func TestInterpretConvertStringToPath(t *testing.T) {
```